client/src/featureform/register.py

```diff
@@ -1342,6 +1354,14 @@ def register_training_set(self,
             owner = owner.name()
         if owner == "":
             owner = self.must_get_default_owner()
+        if isinstance(features,tuple):
+            raise ValueError("Features must be entered as a list")
+        if isinstance(label, list):
+            if len(label) != 1:
+                raise ValueError("Only one label can be used to register a training set")
+            label = (label[0]["name"], label[0]["variant"])
```

Comment:
Put this logic in a label() function instead. They should be able to register mutliple labels, but only call a single one